docs/4.0/trustedclusters.md

```diff
@@ -219,6 +236,17 @@ role name and use reference it to name the local role:
 **NOTE:** The regexp matching is activated only when the expression starts
 with `^` and ends with `$`
 
+
+### Trusted Cluster UI
+For customers using Teleport Pro, they can easily configure _leaf_ nodes using the 
```

Comment:
I don't think we sell Teleport Pro any more so this should reference Enterprise. I also don't think that the trusted cluster UI is an Enterprise-only feature - probably need to double check.

Reply:
Good call, yeah, I checked it's a Ent only feature in 4.0